locations/spiders/mydentist_gb.py

```diff
@@ -7,10 +7,10 @@
 
 
 class MyDentistGBSpider(CrawlSpider, StructuredDataSpider):
+    MYDENTIST = {"brand": "My Dentist", "brand_wikidata": "Q65118035"}
+
     name = "mydentist_gb"
     item_attributes = {
-        "brand": "My Dentist",
-        "brand_wikidata": "Q65118035",
         "country": "GB",
     }
```

Comment:
```suggestion
```

We can actually get the country from the spider name mydentist_**gb** or the domain name, (or geocoding). So this isn't necessary any more.